nvi-commons/src/main/resources/shape.ttl

```diff
@@ -0,0 +1,207 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix : <https://nva.sikt.no/ontology/publication#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:PublicationShape a sh:NodeShape ;
+  sh:targetClass :Publication ;
+  sh:property [
+    sh:path :contributor ;
+    sh:minCount 1 ;
+    sh:nodeKind sh:IRI
+  ] ;
+  sh:property [
+    sh:path :identifier ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string
+  ] ;
+  sh:property [
+    sh:path :modifiedDate ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:dateTime
+  ] ;
+  sh:property [
+    sh:path :abstractText ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string
+  ] ;
+  sh:property [
+    sh:path :isApplicable ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:boolean ;
+    sh:in ( "true"^^xsd:boolean )
+  ] ;
+  sh:property [
+    sh:path :isInternationalCollaboration ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:boolean
+  ] ;
+  sh:property [
+    sh:path :language ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:IRI
+  ] ;
+  sh:property [
+    sh:path :pageCount ;
+    sh:minCount 1 ;
+    sh:maxCount 1
+  ] ;
+  sh:property [
+    sh:path :publicationChannel ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
```

Comment:
```suggestion
    sh:maxCount 2 ;
```

The parser includes all publication channels still, not just the one we use for NVI calculation. Should probably change that, but for now we can have multiple channels.,